planner/cascades/rule_FilterAggregateTranspose.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 PingCAP, Inc.
```

Comment:
Seems that golang recommend file name only containing lowercase?

Reply:
I'll refactor this later when UT is added.